releasenotes/notes/revision-tag-command.yaml

```diff
@@ -1,7 +1,6 @@
 apiVersion: release-notes/v2
 kind: feature
 area: installation
-issue:
```

Comment:
I'm not sure why this file is in the commit, unless it's because the commit is an much earlier commit. Normally when you mark the change as non-facing you wouldn't include a release notes file.

Reply:
Because it caused release-notes_istio to fail and failed to pass verification. So I am directly modifying it here.

Reply:
@brian-avery FYI.

Reply:
That change was correct and if an array has no values, then arrays do sometimes get interpreted as null instead of an array. That said, this file has been in here since April and the release notes schema validation was added in May. Other pull requests continue to pass without any errors. So I'm not sure why it started failing here. That said. Good catch and thanks for fixing it.